Harvard-Stellenbosch-University.csl

```diff
@@ -0,0 +1,309 @@
+    <id>http://csl.mendeley.com/styles/21406231/harvard-stellenbosch-university--2</id>
+    <link href="http://www.zotero.org/styles/harvard-university-of-cape-town" rel="self"/>
+    <link href="http://www.zotero.org/styles/harvard-university-of-gloucestershire" rel="template"/>
+    <link href="http://www.lib.uct.ac.za/wp-content/uploads/2014/02/harvard-uct-2014.pdf" rel="documentation"/>
+    <author>
+      <name>Yusuf Ras</name>
+      <email>yr@sun.ac.za</email>
+    </author>
+    <contributor>
+      <name>Yusuf Ras</name>
+      <uri>http://www.mendeley.com/profiles/yusuf-ras/</uri>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <updated>2015-09-01T08:01:34+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xmlns="http://purl.org/net/xbiblio/csl" xml:lang="en-GB">
+    <terms>
+      <term name="volume" form="short">v.</term>
+      <term name="available at">available</term>
+      <term name="open-quote">�</term>
+      <term name="close-quote">�</term>
+      <term name="open-inner-quote">�</term>
+      <term name="close-inner-quote">�</term>
+    </terms>
+  </locale>
+  <macro name="editor-translator">
+    <choose>
+      <if variable="editor">
+        <names variable="editor">
+          <name and="symbol" initialize-with="." delimiter=", "/>
+          <label form="short" prefix=", " text-case="capitalize-first"/>
+        </names>
+      </if>
+    </choose>
+    <choose>
+      <if variable="translator">
+        <names variable="translator">
+          <label form="verb" text-case="capitalize-first" suffix=" "/>
+          <name and="symbol" delimiter=", "/>
+        </names>
+      </if>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " delimiter-precedes-last="never" et-al-min="9" et-al-use-first="8"/>
+      <label form="short" prefix=" " text-case="capitalize-first"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <choose>
+          <if type="bill book graphic legal_case legislation motion_picture report song webpage" match="any">
+            <text variable="title" font-style="italic"/>
+          </if>
+          <else>
+            <text variable="title" quotes="true" text-case="capitalize-first"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="symbol" delimiter-precedes-last="never" et-al-min="5" et-al-use-first="5" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize-with="." sort-separator=" "/>
+      <et-al suffix="., " font-style="italic"/>
```

Comment:
while possible, don't set a suffix on et al.: the period is there anyway, and the comma should probably be after the name either way, no? So you could just set it after the macro in the citation.

Also, ```el-al-use-first``` is always at least one less than ```et-al-min```. And could you confirm you really want (Smith, Meyer, Jones & Davis et al. 2001)? Four authors before et al seems highly unusual.